main.go

```diff
@@ -0,0 +1,29 @@
+package main
+
+import (
+	"log"
+	"net/http"
+	"os"
+
+	whproxy "github.com/taskcluster/webhooktunnel/proxy"
+)
+
+// starts proxy on a random port on the system
+func main() {
+	proxy := whproxy.NewProxy(whproxy.Config{
+		Logger: log.New(os.Stdout, "proxy", log.Lshortfile),
+	})
+
+	done := make(chan bool, 1)
+	server := &http.Server{Addr: ":9999", Handler: proxy.GetHandler()}
```

Comment:
Let's read the `PORT` from environment variable...
And add a `// TODO: add support config file and TLS` we don't have to do either just yet... but let's leave todo because this is the place we'll do it, right?